src/clj/chocolate/db/core.clj

```diff
@@ -6,11 +6,11 @@
     [mount.core :refer [defstate]]
     [chocolate.config :refer [env]]))
 
-(defstate ^:dynamic *db*
-          :start (conman/connect! {:jdbc-url (env :database-url)})
-          :stop (conman/disconnect! *db*))
-
-(conman/bind-connection *db* "sql/queries.sql")
+;(defstate ^:dynamic *db*
+;          :start (conman/connect! {:jdbc-url (env :database-url)})
+;          :stop (conman/disconnect! *db*))
+;
+;(conman/bind-connection *db* "sql/queries.sql")
 
 
 (extend-protocol jdbc/IResultSetReadColumn
```

Comment:
Do we need this since we've removed the db?

Reply:
I left it in under the assumption that we would one day start using the DB again for some reason.